decidim-core/app/presenters/decidim/admin_log/user_presenter.rb

```diff
@@ -25,10 +26,40 @@ def action_string
 
       def i18n_params
         super.merge(
-          role: I18n.t("models.user.fields.roles.#{user_role}", scope: "decidim.admin")
+          role: I18n.t("models.user.fields.roles.#{user_role}", scope: "decidim.admin"),
+          blocked_count: blocked_users.count,
+          unblocked_count: unblocked_users.count,
+          unreported_users_count: unreported_users.count,
+          hidden_count: hidden_content.count,
+          unhidden_count: unhidden_content.count,
+          unreported_count: unreported_content.count
         )
       end
 
+      def blocked_users
+        action_log.extra.dig("extra", "blocked")&.values || []
+      end
+
+      def unblocked_users
+        action_log.extra.dig("extra", "unblocked")&.values || []
+      end
+
+      def unreported_users
+        action_log.extra.dig("extra", "unreported")&.values || []
+      end
+
+      def hidden_content
+        action_log.extra.dig("extra", "reported_content")&.values || []
+      end
+
+      def unhidden_content
+        action_log.extra.dig("extra", "reported_content")&.values || []
+      end
+
+      def unreported_content
+        action_log.extra.dig("extra", "reported_content")&.values || []
+      end
```

Comment:
```suggestion
      def reported_content
        action_log.extra.dig("extra", "reported_content")&.values || []
      end
```

I see a lot of repeated code, let's just use one method an reuse it

Reply:
Done